set.mm

```diff
@@ -437504,10 +437504,10 @@ orthogonal vectors (i.e. whose inner product is 0) is the sum of the
 /* Elementary  geometry (extension)  */
 htmldef "LineM" as 'LineM';
  althtmldef "LineM" as 'Line<sub>M</sub>';
- latexdef "LineM" as "{\rm Line}_M";
+ latexdef "LineM" as "{\mathrm{Line}_M}";
```

Comment:
The outside brackets can be removed here, I guess.

Reply:
By the way: anyone knows why the "suggestion" feature is unavailable ? It would be pretty handy here: we could even apply the suggestion and merge without waiting.

Reply:
@benjub - you have write access to this repo, so you *should* be able to the suggestion feature as far as I know.

Reply:
```suggestion
 latexdef "LineM" as "\mathrm{Line}_M";
```

Reply:
Just made the suggestion after several different trials (the button was inactive when in "Review" mode for a reason I cannot pin down).

Reply:
Okay, will do!